modules/prometheus-exporter_oracledb/conf/readme.yaml

```diff
@@ -0,0 +1,83 @@
+documentations:
+  - name: Prometheus Exporter for oracledb
+    url: https://github.com/iamseth/oracledb_exporter
+
+source_doc: |
+  The detectors of this module uses defaults metrics from the [oracledb exporter] (https://github.com/iamseth/oracledb_exporter) and optionnaly custom metrics from custom templates which have to be specified during the deployment.  
+  Check its documentation to install and configure it appropriately with your Oracle database host.
+
+  by default in this module, only the metric oracledb_up is used with the default.metrics related to the [oracledb exporter]. 
```

Comment:
to remove, it is not the default it is always true

Reply:
done